examples/ethers_base/test/tests.ts

```diff
@@ -7,176 +7,138 @@
-                };
-            }
-        },
-        {
-            name: '/address-balance canister',
-            test: async () => {
-                const response = await fetch(`${origin}/address-balance`, {
-                    method: 'POST',
-                    headers: [['Content-Type', 'application/json']],
-                    body: JSON.stringify({
-                        address: canisterAddress
-                    })
-                });
-                const responseJson = await response.json();
-
-                return {
-                    Ok: BigInt(responseJson.__bigint__) === 100_000_000_000_000n
-                };
-            }
-        },
-        {
-            name: '/transfer-from-canister to caller wallet',
-            test: async () => {
-                const response = await fetch(
-                    `${origin}/transfer-from-canister`,
-                    {
-                        method: 'POST',
-                        headers: [['Content-Type', 'application/json']],
-                        body: JSON.stringify({
-                            to: callerAddress,
-                            value: '.000000000000000007'
-                        })
-                    }
-                );
-                const responseText = await response.text();
-
-                return {
-                    Ok: responseText.startsWith('transaction sent with hash:')
-                };
-            }
-        },
-        {
-            name: '/address-balance canister',
-            test: async () => {
-                const response = await fetch(`${origin}/address-balance`, {
-                    method: 'POST',
-                    headers: [['Content-Type', 'application/json']],
-                    body: JSON.stringify({
-                        address: canisterAddress
-                    })
-                });
-                const responseJson = await response.json();
-
-                return {
-                    Ok: BigInt(responseJson.__bigint__) < 100_000_000_000_000n
-                };
-            }
-        },
-        {
-            name: '/address-balance caller',
-            test: async () => {
-                const response = await fetch(`${origin}/address-balance`, {
-                    method: 'POST',
-                    headers: [['Content-Type', 'application/json']],
-                    body: JSON.stringify({
-                        address: callerAddress
-                    })
-                });
-                const responseJson = await response.json();
-
-                return {
-                    Ok: BigInt(responseJson.__bigint__) === 7n
-                };
-            }
-        }
-    ];
+                }
+            );
+            const responseText = await response.text();
+
+            expect(responseText).toMatch('transaction sent with hash:');
+        });
+
+        wait('for funds to be available', 30_000);
```

Comment:
This is weird that this needed to be added... any ideas what's going on here?

Reply:
Let's take a look

Reply:
we looked at it and decided to add a 10 second wait after each transfer to be sure that it will work on whatever computer. It is interesting to note that these same tests were failing on my computer even on the main branch